tests/behat/fixtures/civictheme_migrate.node_civictheme_page_3.json

```diff
@@ -0,0 +1,278 @@
+[
+  {
+    "id": "77d61781-0657-4f9a-9d12-173ea03eae91",
+    "title": "[TEST] Migrated Page Content 1",
+    "alias": "/migrated/page-content-1",
+    "summary": "<span>Designed by government, for government. Our goal is to help you deliver and maintain secure, modern and user-friendly websites. GovCMS is community-based, but don't just take our word for it.</span>",
+    "topics": "[TEST] Topic 1,[TEST] Topic 2,[TEST] Topic 3,[TEST] Topic 4",
+    "thumbnail": ["c97a9b08-f3b0-477b-97f7-8b61f9d4a527"],
+    "vertical_spacing": "top",
+    "hide_sidebar": true,
+    "show_last_updated_date": true,
+    "last_updated_date": "1 July 2020",
+    "banner": {
+      "type": "container",
+      "children": [
+        {
+          "theme": "dark",
+          "title": "[TEST] Banner title - Migrated Page Content 1",
+          "blend_mode": "darken",
+          "featured_image": ["7fdce6fd-3bcb-4ffa-b349-2a6eb0b049c4"],
+          "background": ["7966cf0a-598d-4da2-8443-837fccd46109"],
+          "hide_breadcrumb": true
+        }
+      ]
+    },
+    "content": {
+      "type": "container",
+      "children": [
+        {
+          "text_content": {
+            "format": "rich_text",
+            "value": "<p>[Content migration] - Product and Services Guide</p>"
```

Comment:
@govindmaloo 
Can you please use descriptive content. `Product and Services Guide` does not describe where this text will appear in the migrated manual list. It will be problematic to debug this later if we do not know where the value is expected to appear.